src/styles/blocks/container.scss

```diff
@@ -0,0 +1,19 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+  width: 4 * $cardWidth + 3 * $gap;
+  padding-top: 51px;
+
+  & > :nth-child(n) {
+    margin-right: $gap;
+  }
+
+  & > :nth-child(4n) {
+    margin-right: 0;
+  }
+
+  & > :nth-child(n + 5) {
+    margin-top: 48px;
+  }
```

Comment:
same

Reply:
container and card are  two different BEM blocks(card is not element of container). &__card :nth-child(n) doesnt work in my code. when I set just  & :nth-child(n), styles implement to all levels children of card class.  Similar way:  & .card :nth-child(n) -the same case, all levels children accept these styles settings.